python/oneflow/test/modules/test_consistent_sparse.py

```diff
@@ -0,0 +1,53 @@
+"""
+Copyright 2020 The OneFlow Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import unittest
+from oneflow.test_utils.automated_test_util import *
+import oneflow as flow
+import oneflow.unittest
+
+
+@autotest(n=1, check_graph=False)
+def _test_embedding(test_case, ndim, placement, sbp):
+    emb_size = random() * 8
+    emb_dim = random() * 8
+    emb_shape = [emb_size, emb_dim]
+    emb_sbp = [flow.sbp.broadcast for _ in range(len(sbp))]
+
+    idx_shape = [random(high=4) * 8 for i in range(ndim)]
+
+    weight = random_tensor(2, *emb_shape)
+    indices = random_tensor(len(idx_shape), *idx_shape, low=0, high=emb_size, dtype=int).to_global(
+        placement=placement, sbp=emb_sbp
+    )
+    embedding = torch.nn.Embedding(emb_size, emb_dim, _weight=weight).to_global(
+        placement=placement, sbp=sbp
+    )
```

Comment:
```suggestion
    embedding = torch.nn.Embedding(emb_size, emb_dim, _weight=weight)
    embedding.oneflow.to_global(placement=placement, sbp=sbp)
```

这里可能是自动测试框架的 bug，这里拆开写就可以了

Reply:
好的

Reply:
这个应该不能算是bug，因为torch根本就没有to_global这个接口。

Reply:
random_tensor 是可以 to_global 的，其实框架里处理了 to_global 和 to_local 的情况（ https://github.com/Oneflow-Inc/oneflow/blob/f9265eba3a9e3854e7e5e6fb5af434fd4c24fe91/python/oneflow/test_utils/automated_test_util/torch_flow_dual_object.py#L825 ），只是没处理正确，导致这里本来是 to cuda，但结果是 cpu。